cetmix_tower_ovh/demo/demo_data.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <!-- ===============================================
+    Keys
+    ================================================ -->
+    <record id="ovh_application_key" model="cx.tower.key">
+        <field name="name">OVH Application Key</field>
+        <field name="reference">ovh_application_key</field>
+        <field name="key_type">s</field>
+        <field name="secret_value" />
+    </record>
+
+    <record id="ovh_application_secret" model="cx.tower.key">
+        <field name="name">OVH Application Secret</field>
+        <field name="reference">ovh_application_secret</field>
+        <field name="key_type">s</field>
+        <field name="secret_value" />
+    </record>
+
+    <record id="ovh_consumer_key" model="cx.tower.key">
+        <field name="name">OVH Consumer Key</field>
+        <field name="reference">ovh_consumer_key</field>
+        <field name="key_type">s</field>
+        <field name="secret_value" />
+    </record>
+
+    <record id="ovh_zone_name" model="cx.tower.key">
+        <field name="name">OVH Zone Name</field>
+        <field name="reference">ovh_zone_name</field>
+        <field name="key_type">s</field>
+        <field name="secret_value" />
+    </record>
+
+    <!-- ===============================================
+    Variables
+    ================================================ -->
+    <record id="variable_ovh_endpoint" model="cx.tower.variable">
+        <field name="name">OVH Endpoint</field>
+        <field name="reference">ovh_endpoint</field>
+    </record>
+
+    <!-- ===============================================
+    Global Variable values
+    ================================================ -->
+    <record id="global_value_ovh_endpoint" model="cx.tower.variable.value">
+        <field name="variable_id" ref="variable_ovh_endpoint" />
+        <field name="value_char">ovh-eu</field>
+    </record>
+
+    <!-- ===============================================
+    Commands
+    ================================================ -->
+    <!-- prettier-ignore-start -->
+    <record id="command_list_ovh_servers" model="cx.tower.command">
+        <field name="name">Register OVH DNS Record</field>
+        <field name="action">python_code</field>
+        <field name="code">
+# ----------------
+# Tower properties
+# ----------------
+
+# Ensure your domain matches your DNS Zone!
+instance_url = {{ tower.server.url }}
+ovh_endpoint = {{ ovh_endpoint }}
+ovh_application_key = #!cxtower.secret.ovh_application_key!#
+ovh_application_secret = #!cxtower.secret.ovh_application_secret!#
+ovh_consumer_key = #!cxtower.secret.ovh_consumer_key!#
+ovh_zone_name = #!cxtower.secret.ovh_zone_name!#
+
+# ----------------
+# Request settings
+# ----------------
+
+path = "/domain/zone/"+ovh_zone_name+"/record"
+sub_domain = tldextract.extract(instance_url).subdomain
+
+# Send request to Ovh
+exit_code = 0
+message = f"subDomain {sub_domain} exists."
```

Comment:
Unfortunately Odoo safe_eval doesn't support f-strings, at least in version 14.0. So val1 + val2 is the only working option (